eslint.shared.config.mjs

```diff
@@ -37,6 +37,7 @@ export default [
         ...globals.browser,
         ...globals.node,
         ...globals.webextensions,
+        NodeJS: true,
```

Comment:
This is the case for `builders`, `cli`, `schematics`, `test` and `scripts`.
It may be very valuable to forbid `Node` in the other cases.
(same comment for `globals.node` and reverse comment for `globals.browser`)

Reply:
It's only for us, it's not the shared config

Reply:
yes, my comment was for us :)

Reply:
I take the point 
I will do it in another PR

Reply:
https://github.com/AmadeusITGroup/otter/pull/2361